addons/payment_paypal/tests/test_paypal.py

```diff
@@ -47,18 +53,23 @@ def _get_expected_values(self):
 
     def test_redirect_form_values(self):
         tx = self._create_transaction(flow='redirect')
-        with mute_logger('odoo.addons.payment.models.payment_transaction'):
-            processing_values = tx._get_processing_values()
+        with patch(
+            "odoo.addons.payment.utils.generate_access_token",
+            new=self._generate_test_access_token,
+        ):
+            with mute_logger('odoo.addons.payment.models.payment_transaction'):
+                processing_values = tx._get_processing_values()
```

Comment:
These two context managers don't need to be one inside another; they can be chained like this: `with patch(...), patch(...):